### DISCUSSION TITLE
Account Suspended and no reply from Support

### DISCUSSION THREAD

**Discussion by Venchislav** [community/community#126554] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, I'm not sure if I selected the right discussion label, but anyway.
On May 11, 2024, my GitHub was suspended and I lost the ability to share my code with the community.
I followed the GitHub guide (password change, two-factor authentication) and opened a Reinstatement request.
(ticket number #2780328). 3 weeks have passed since then and I have not received any feedback.
I remain patient (as I know GitHub support is busy with various requests).
I checked other discussions on this topic (Reddit, discussions), but the maximum time period I've heard of is 2 weeks.
I just feel a little uncomfortable with my account being disabled since all my code can't be shared with other users.

I will be glad to any response.
Hope you can help

**Comment by RManLuo** [comment]
Hi, I am facing the same issue. Did your problem solve right now?
